backend/src/api/users/users.api.ts

```diff
@@ -32,6 +33,15 @@ const initUsersApi = ({ apiRouter, userService }: Args): Router => {
     }),
   );
 
+  userRouter.get(
+    UsersApiPath.POPULAR,
+    handleAsyncApi(async (req, res) => {
+      return res
+        .json(await userService.getPopular(req.query as unknown as UserPopularLoadFilter))
```

Comment:
```suggestion
        .json(await userService.getPopular(req.query as UserPopularLoadFilter))
```

Reply:
`Conversion of type 'ParsedQs' to type 'UserPopularLoadFilter' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'limit' is missing in type 'ParsedQs' but required in type 'UserPopularLoadFilter'.`